GameOfLife/Main/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿
+
+using GameOfLife;
+
+public class Program
+{
+    static public void Main(string[] args)
```

Comment:
Typically, Main method of program.cs is used only as an entry point to your application and should not contain much logic at all. All the code you have in here should be moved to a new class, and then in here, in the main method, you can just start you app by calling the new class.